src/stores/SettingsStore.ts

```diff
@@ -44,6 +52,8 @@ export const useSettingsStore = defineStore('SettingsStore', () => {
       showNotifications: false,
       bugzillaApiKey: '',
       jiraApiKey: '',
+      affectsPerPage: 10,
+      trackersPerPage: 10,
```

Comment:
Should we use the `defaultValues` variable for this?

```typescript
function $reset() {
  settings.value = defaultValues; // maybe {...defaultValues} in case there's some object reference
}
```